ph5/utilities/ph5validate.py

```diff
@@ -9,11 +9,15 @@
 import argparse
 import logging
 import re
-import subprocess
 import sys
+import os
+
+import subprocess
+
 from ph5.core import ph5api
+from ph5 import LOGGING_FORMAT
 
-PROG_VERSION = "2019.29"
+PROG_VERSION = "2019.228"
 LOGGER = logging.getLogger(__name__)
```

Comment:
I have to fix the logging because I got error when running unittest:
IOError: [Errno 2] No such file or directory: '/Users/field/Documents/GIT/PH5/ph5/test_data/tmpp37b3O/ph5_validate.log'
Though we already clean up all the Filehandlers in logging. You can open the error file for more detail.
[PR373_logging error.txt](https://github.com/PIC-IRIS/PH5/files/4599498/PR373_logging.error.txt)
I found 2 solutions for this. 
+ One is changing test_base.LogTestCase so that it won't set logger.propagate to 1 (keep it 0 as set in ph5.__init__) as I suggest in PR #396 . However you don't want to remove that from test_base for any reason.
+ Another solution is fixing logging setup to be similar with other tools like I am doing here. If you don't want to fix it in this PR. Can I fix it in another PR and merge it before this PR?

Reply:
Can you please debug this and describe the problem?

Reply:
Option 2 sounds like the way to go, make the general case work with this case too.

Reply:
I don't have to change the logging setup anymore after I change the way of adding data from using tools for adding raw data to only adding metadata with test_base.kef_to_ph5(). But maybe I will create an issue and a PR to modify the logging for ph5validate later.

Reply:
So this can be changed to the correct way?
Please when I ask a question, answer, don't make more changes. Answer the question.

Reply:
You said :

> this isn't the PR to fix it.

That why I changed back.